core/unboundmethod/private_spec.rb

```diff
@@ -18,4 +18,11 @@
       obj.method(:my_private_method).unbind.private?.should == true
     end
   end
+
+  ruby_version_is "3.2" do
+    it "has been removed" do
+      obj = UnboundMethodSpecs::Methods.new
+      obj.method(:my_private_method).should_not.respond_to?(:private?)
```

Comment:
**major:** Without `unbind` it will test an instance of `Method`, not `UnboundMethod`
```suggestion
      obj.method(:my_private_method).unbind.should_not.respond_to?(:private?)
```